rna-transcription.json

```diff
@@ -0,0 +1,96 @@
+{
+   "#": [
+     "Language implementations vary on the issue of invalid input data.",
+     "A language may elect to simplify this task by only presenting valid",
+     "test cases.  For languages handling invalid input data as",
+     "error conditions, invalid test cases are included here and are",
+     "indicated with an expected value of null.  Note however that null is",
+     "simply an indication here in the JSON.  Actually returning null from",
+     "a rna-transcription function may or may not be idiomatic in a languge.",
+     "Language idioms of errors or exceptions should be followed.",
+     "Alternative interpretations such as ignoring excess length at the end",
+     "are not represented here."
+   ],
+   "cases": [
+      {
+         "description": "rna complement of cytosine is guanine",
+         "dna" : "C", 
+         "expected": "G"
+      },
+      {
+         "description": "rna complement of guanine is cytosine",
+         "dna": "G",
+         "expected": "C"
+      },
+      {
+         "description": "rna complement of thymine is adenine",
+         "dna": "T",
+         "expected": "A"
+      },
+      {
+         "description": "rna complement of adenine is uracil",
+         "dna": "A",
+         "expected": "U"
+      },
+      {
+         "description": "rna complement",
+         "dna": "ACGTGGTCTTAA",
+         "expected": "UGCACCAGAAUU"
+      },
+         {
+         "description": "dna complement of cytosine is guanine",
+         "rna" : "C", 
+         "expected": "G"
+      },
+      {
+         "description": "dna complement of guanine is cytosine",
+         "rna": "G",
+         "expected": "C"
+      },
+      {
+         "description": "dna complement of uracil is adenine",
+         "rna": "U",
+         "expected": "A"
+      },
+      {
+         "description": "dna complement of adenine is thymine",
+         "rna": "A",
+         "expected": "T"
+      },
+      {
+         "description": "dna complement",
+         "rna": "UGAACCCGACAUG",
+         "expected": "ACTTGGGCTGTAC"
+      },
+      {
+         "description": "dna raises argument error",
```

Comment:
Description should be about being invalid, not about raising a specific error (language-dependent implementation).